R/pkg/tests/fulltests/test_context.R

```diff
@@ -21,10 +21,11 @@ test_that("Check masked functions", {
   # Check that we are not masking any new function from base, stats, testthat unexpectedly
   # NOTE: We should avoid adding entries to *namesOfMaskedCompletely* as masked functions make it
   # hard for users to use base R functions. Please check when in doubt.
-  namesOfMaskedCompletely <- c("cov", "filter", "sample", "not")
+  namesOfMaskedCompletely <- c("cov", "filter", "sample", "not", "sequence")
```

Comment:
@felixcheung, I remember we discourage to exclude this?

Reply:
yes, we should definitely try to avoid adding here (as in the comment above) unless we absolutely have to.

in this case, base::sequence can be commonly used - we really need a way to avoid breaking it.
for example, could you try setting the generic as such:

`setGeneric("sequence", function(...) { standardGeneric("sequence") })`

Reply:
Thanks @HyukjinKwon @felixcheung for your review. I will change this.

Reply:
thanks, this is a bit tricky because of the way tests work, could you do one check - install the package locally (`R CMD INSTALL --library=lib pkg/`) and then load the package from lib, in R, `library(SparkR, lib.loc='pathto/lib')`
then try calling base::sequence and SparkR::sequence

also, please add a test for base::sequence too - we have that for other functions